src/mame/atari/skydiver.cpp

```diff
@@ -417,4 +447,5 @@ ROM_END
  *
  *************************************/
 
-GAMEL(1978, skydiver, 0, skydiver, skydiver, skydiver_state, empty_init, ROT0, "Atari", "Sky Diver", MACHINE_SUPPORTS_SAVE, layout_skydiver )
+GAMEL(1978, skydiver,  0,        skydiver, skydiver, skydiver_state, empty_init, ROT0, "Atari", "Sky Diver (set 1)", MACHINE_SUPPORTS_SAVE, layout_skydiver )
+GAMEL(1978, skydivera, skydiver, skydiver, skydiver, skydiver_state, empty_init, ROT0, "Atari", "Sky Diver (set 2)", MACHINE_SUPPORTS_SAVE, layout_skydiver )
```

Comment:
Who should be credited for this additional set?

Reply:
That one was from andysarcade